utils.py

```diff
@@ -453,3 +454,20 @@ def get_all_language_codes_and_names():
 def unescape_encoded_uri_component(escaped_string):
     """Unescape a string that is encoded with encodeURIComponent."""
     return urllib.unquote(escaped_string).decode('utf-8')
+
+
+ASSET_DIR_PREFIX = None
+def get_asset_dir_prefix():
+    """Returns prefix for asset directory depending whether dev or prod.
+    It is used as a prefix in urls for images, css and script files.
+    """
+    global ASSET_DIR_PREFIX # pylint: disable=global-statement
+    if ASSET_DIR_PREFIX is None:
```

Comment:
Sorry, what do you mean by "auto server restarts"? Do you mean restarts on the local machine by the developer?

I'm not sure what caused it, tbh, but I remember fiddling around with IS_MINIFIED in feconf and the env variable in app.yaml to try and figure out why things weren't serving from the right directory, and the issue was that the asset dir prefix had already been set to '' even though I was running with --prod_env.